dipy/workflows/segment.py

```diff
@@ -368,3 +372,130 @@ def run(
             new_sft = StatefulTractogram(bundle, sft, Space.RASMM)
             save_tractogram(new_sft, out_bundle, bbox_valid_check=False)
             logging.info(out_bundle)
+
+
+class ClassifyTissueFlow(Workflow):
+    @classmethod
+    def get_short_name(cls):
+        return "extracttissue"
+
+    def run(
+        self,
+        input_files,
+        bvals_file=None,
+        method=None,
+        wm_threshold=0.5,
+        b0_threshold=50,
+        low_signal_threshold=50,
+        nclass=3,
+        beta=0.1,
+        tolerance=1e-05,
+        max_iter=100,
+        out_dir="",
+        out_tissue="tissue.nii.gz",
+        out_pve="tissue_pve.nii.gz",
+    ):
+        """Extract tissue from a volume.
+
+        Parameters
+        ----------
+        input_files : string
+            Path to the input volumes. This path may contain wildcards to
+            process multiple inputs at once.
+        bvals_file : string, optional
+            Path to the b-values file. Required for 'dam' method.
+        method : string, optional
+            Method to use for tissue extraction. Options are:
+                - 'hmrf': Markov Random Fields modeling approach.
+                - 'dam': Directional Average Maps, proposed by :footcite:p:`Cheng2020`.
+
+            'hmrf' method is recommended for T1w images, while 'dam' method is
+            recommended for DWI images.
```

Comment:
Should we also add that the DWI image should be multi shell?

Reply:
Thanks, added!